x-pack/docs/en/rest-api/security/update-api-key.asciidoc

```diff
@@ -1,5 +1,277 @@
 [role="xpack"]
 [[security-api-update-api-key]]
-=== Update API key information API
+=== Update API key API
 
-coming::[8.4.0]
+++++
+<titleabbrev>Update API key</titleabbrev>
+++++
+
+[[security-api-update-api-key-request]]
+==== {api-request-title}
+
+`PUT /_security/api_key/<id>`
+
+[[security-api-update-api-key-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have at least the `manage_own_api_key` cluster privilege.
+Users can only update API keys that they created or that were granted to them.
+To update another user's API key, use the <<run-as-privilege,`run_as` feature>>
+to submit a request on behalf of another user.
+
+IMPORTANT: It's not possible to use an API key as the authentication credential for this API.
+To update an API key, the owner user's credentials are required.
+
+[[security-api-update-api-key-desc]]
+==== {api-description-title}
+
+Use this API to update API keys created by the <<security-api-create-api-key,create API Key>> or <<security-api-grant-api-key,grant API Key>> APIs.
+It's not possible to update expired API keys, or API keys that have been invalidated by <<security-api-invalidate-api-key,invalidate API Key>>.
+
+This API supports updates to an API key's access scope, i.e., its permissions, and its metadata.
+The access scope of an API key is derived from the <<security-api-update-api-key-api-key-role-descriptors,`role_descriptors`>> you specify in the request, and a snapshot of the owner user's permissions at the time of the request.
```

Comment:
We can be more explicit here. Instead of saying "is derived", we can just say "is an intersection of ...".

Reply:
I don't think "intersection of" is technically correct here without the disclaimer that when `role_descriptors` is empty (or not assigned) it's not the intersection. I went for "derived" because it side-steps that problem, and references the role desc section where we accurately describe how the permissions are derived.